utils/data_processing/recode_csv.py

```diff
@@ -0,0 +1,39 @@
+"""
+Recode the levels of the factors in a csv file.
+"""
+from typing import List, Dict
+import pandas as pd
+
+
+def main(path: str, levels_eq: List[Dict], sep=';', decimal=','):
+    # Load the csv file
+    df = pd.read_csv(filepath, sep=sep, decimal=decimal)
+    # Recode the variables
+    columns = df.columns
+    for level in levels_eq:
+        col = columns[level.get('index')]
+        df[col] = df[col].map(level.get('map'))
+    # Save the new df to csv file
+    new_path = path.replace('.csv', '_recoded.csv')
+    df.to_csv(new_path, sep=sep, decimal=decimal, index=False)
+
+
+if __name__ == "__main__":
+    filepath = 'csv/polyethylene_film.csv'
```

Comment:
this is hardcoded for a certain csv file. If we want this script to be useful we should create a CLI for it like we did for `csv2json.py`, shouldn't we? (using click)